tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "CommonJS",
```

Comment:
Do we need to do this?

Reply:
https://www.typescriptlang.org/tsconfig#module

Enable running as `ts-node api/clober.ts`